docs/guides/04-data-driven-visualizations-part-1.md

```diff
@@ -0,0 +1,184 @@
+
+ Sometimes, the data has [outliers](https://en.wikipedia.org/wiki/Outlier): features with data that is very far away from the norm. In these cases, we may want to ignore them when computing the ramp. This can be easily done by manually setting the second and third parameters of linear to the minimum and maximum values of the data range we are interested.
+
+Let's see an example with *implicit casts* and explicit linear range.
+
+<div class="example-map" style="margin: 20px auto !important">
+    <iframe
+        id="population-density-basic"
+        src="/developers/carto-vl/examples/maps/guides/ramp/population-density-basic.html"
+        width="100%"
+        height="1000"
+        frameBorder="0">
+    </iframe>
+    <a href="/developers/carto-vl/examples#example-population-density---basic">View my source code!</a>
+</div>
+
+
+##### Classifying numerical properties
+
+ Usage of [`linear`](https://carto.com/developers/carto-vl/reference/#cartoexpressionslinear) reduces the loss of precision compared to the usage of classifiers. However, correctly classified data makes easier to detect patterns and improve the perception of the data, since it is difficult to perceive small differences in color or size, which can arise when using [`linear`](https://carto.com/developers/carto-vl/reference/#cartoexpressionslinear).
+
+There are multiple classifying methods (quantiles, equal intervals...) and the classification can be applied to two different samples:
+- The entire dataset. `global*` classifiers will apply the classification to all source data. Ignoring filters or the presence of each feature in the viewport.
+- Viewport data. `viewport*` classifiers will apply the classification only to the features that are on the viewport. This includes filtering by the `filter:` styling property and filtering by checking that the feature is within the region covered by the screen at each moment. Changes in the view (map center/map zoom) will trigger an automatic re-computation of the classification.
+
+On top on that, you can also classify the data by a fixed list of breakpoints with the [`buckets()`](https://carto.com/developers/carto-vl/reference/#cartoexpressionsbuckets) function. For example, the expression `buckets($price, [10, 200])` will classify the features by its price into 3 different categories (buckets): the features that have a price less than 10, the features that have a price between 10 and 200, and the features that have a price higher than 200. It's important to note that there is always one more category than breakpoint. The `buckets` function can also be used with categorical inputs, we'll see that on the [next section](#One_to_one_mapping._One_category_-_one_color.).
```

Comment:
" there is always one more category than breakpoint" -> " there's always one more category than number of breakpoints"